rhobs/alerting/data_plane/prometheus.gitops_statefulsets_alerts.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-statefulsets-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_statefulsets_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsStatefulSetsHealthErrors
+      expr: |
+        (
+          sum(kube_statefulset_status_replicas_ready{namespace="gitops-service-argocd"})
+          by (source_cluster)
+        )
+        /
+        (
+          sum(kube_statefulset_replicas{namespace="gitops-service-argocd"})
+          by (source_cluster)
+        ) < 0.95
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
```

Comment:
I suggest adding the source_cluster to the summary/description

Reply:
Does it mean that I always need to have source_cluster in all the tests?

Reply:
I think it will be empty otherwise. It might be a little burden to set it in the test, but it will make the alert more useful. Otherwise, you won't have a quick way to tell which cluster is alerting.

Reply:
I added it in all test cases, as you said it makes alert more useful, so better to add. PTAL.